useful_package/__init__.py

```diff
@@ -1,7 +1,8 @@
-<<<<<<< HEAD
-#!/usr/bin/env python3
 
-from module_b import hyperbola
-from module_a import polynom_3
-#print('all went up to an end')
->>>>>>> module_a_dev
```

Comment:
delete the functional lines when performing the conflict resolution, not manually after.